applications/tari_merge_mining_proxy/README.md

```diff
@@ -21,25 +25,17 @@ Follow build instructions for XMRig here:
 Then create a configuration for XMRig using the wizard here:
 `https://xmrig.com/wizard`
 
-When using the wizard you need to use `Add daemon` and the following public stagenet wallet address can be used:
-`55LTR8KniP4LQGJSPtbYDacR7dz8RBFnsfAKMaMuwUNYX6aQbBcovzDPyrQF9KXF9tVU6Xk3K8no1BywnJX6GvZX8yJsXvt`
-
-Take note of the `Host` and `Port` that was specified in the configuration.
+When using the wizard you need to use `Add daemon`. The following public stagenet wallet address can be used (or you can use your own stagenet wallet address):
+`55LTR8KniP4LQGJSPtbYDacR7dz8RBFnsfAKMaMuwUNYX6aQbBcovzDPyrQF9KXF9tVU6Xk3K8no1BywnJX6GvZX8yJsXvt`. The `Host` and `Port` fields can be filled in according to the 
+value of `proxy_host_address` that was previously noted down.
 
-You will put the resulting `config.json` file together with the built binaries for XMRig.
+Once you have finished the wizard, you will put the resulting `config.json` file together with the built binaries for XMRig.
 
-Start XMRig from the terminal using `./xmrig`
+Start XMRig from the terminal using `./xmrig`.
 
 ### Tari Merge Mining Proxy
+Then run the Tari Merge Mining Proxy with `cargo run --bin tari_merge_mining_proxy`.
 
-In `main.rs` of the Tari merge mining proxy the following constants can be changed to reflect the above configuration:
-```Rust
-const MONEROD_URL: &str = "monero-stagenet.exan.tech:38081"; // To connect to the Monero Public Stagenet.
-const TARI_GRPC_URL: &str = "http://127.0.0.1:18142"; // To connect to the Tari Base Node.
-const LOCALHOST: &str = "127.0.0.1:7878"; // For XMRig to connect.
-```
+The address of the stagenet daemon you are connecting to can be changed if needed in the `config.toml`, the relevant variable is `monerod_url`.
```

Comment:
Changed to what? i.e. How do you find out what the right url is?

Also, it's `monerod_url` _in the ridcully config section_, i.e. under `[merge_mining_proxy.ridcully]`

Reply:
Have added these details.